lib/foreman_leapp/engine.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+require 'foreman_ansible'
+require 'foreman_remote_execution'
```

Comment:
this fixes https://github.com/theforeman/foreman_leapp/issues/120

(technically, I guess, `foreman_ansible` would be enough, but I felt like being explicit here)